ivy/functional/frontends/__init__.py

```diff
@@ -3,3 +3,87 @@
 from . import jax
 from . import torch
 from . import tensorflow
+import importlib
+
+latest_version = {"torch": "1.12", "tensorflow":"2.9.0", "numpy":"1.23.2", "jax":"0.3.16"}
```

Comment:
Do you think it is necessary for us to manually track the latest versions of each framework? Seems like a lot of manual maintenance effort.